bokehjs/test/unit/models/tools/gestures/tap_tool.ts

```diff
@@ -130,7 +158,11 @@ describe("TapTool", () => {
 
     it("and not trigger on 'doubletap' event when didn't hit a glyph", async () => {
       let called = false
-      const callback: TapToolCallback = {execute() { called = true }}
+      const callback: TapToolCallback = {
+        execute() {
+          called = true
+        },
+      }
```

Comment:
This does not seem to fix any under-bracing? I personally find the single-line version dramatically better. Do you expect the callback body to become much more complicated in the future?

Reply:
I actually resolved this in PR #13451.